client/js/app.js

```diff
@@ -36,6 +36,7 @@ import tabs from './components/tabs';
 import {eventbriteTicketStatus} from './components/eventbrite-ticket-status';
 import newsletterSignup from './components/newsletter-signup';
 import {createMaps} from './components/map';
+import {onWindowResizeDebounce$} from './utils/dom-events';
```

Comment:
I was wondering about the cycle stuff yesterday – if/when we get rid of the sticky-digital-story-nav, should we think about getting rid of cycle? It's a fair chunk of code that we've added for very little gain. And we could use e.g. lodash for debounce/throttle.

Only mentioning here while I remember; not suggesting it should be anything to do with this PR.

Reply:
👍